src/app/form/form.component.ts

```diff
@@ -1,15 +1,67 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, HostListener, OnInit } from '@angular/core';
+import { FormBuilder, FormControl, FormGroup, Validators } from '@angular/forms';
 
 @Component({
   selector: 'app-form',
   templateUrl: './form.component.html',
   styleUrls: ['./form.component.scss']
 })
 export class FormComponent implements OnInit {
+  US_STATES = ['Alabama','Alaska','American Samoa','Arizona','Arkansas','California','Colorado','Connecticut','Delaware',
+  'District of Columbia','Federated States of Micronesia','Florida','Georgia','Guam','Hawaii','Idaho','Illinois','Indiana',
+  'Iowa','Kansas','Kentucky','Louisiana','Maine','Marshall Islands','Maryland','Massachusetts','Michigan','Minnesota',
+  'Mississippi','Missouri','Montana','Nebraska','Nevada','New Hampshire','New Jersey','New Mexico','New York','North Carolina',
+  'North Dakota','Northern Mariana Islands','Ohio','Oklahoma','Oregon','Palau','Pennsylvania','Puerto Rico','Rhode Island',
+  'South Carolina','South Dakota','Tennessee','Texas','Utah','Vermont','Virgin Island','Virginia','Washington',
+  'West Virginia','Wisconsin','Wyoming'];
+  contactInfoFormGroup = new FormGroup({
+    firstName: new FormControl('', [Validators.required, Validators.maxLength(4)]),
+    lastName: new FormControl('', [Validators.required]),
+    address: new FormControl('', [Validators.required]),
+    city: new FormControl('', [Validators.required]),
+    state: new FormControl('', [Validators.required]),
+    zipCode: new FormControl('', [Validators.required]),
+    emailAddress: new FormControl('', [Validators.required]),
+    homePhone: new FormControl('', [Validators.required]),
+    age: new FormControl('', [Validators.required]),
+    gender: new FormControl('', [Validators.required])
+  });
+
+  contactInfo = {
+    firstname: '',
+    lastname: ''
+  };
+
+  orientationType: 'vertical' | 'horizontal' = 'horizontal';
+  @HostListener('window:resize') onWindowResize() {
```

Comment:
what does this do?

Reply:
Is this for the form?

Reply:
Oh is see it is for the stepper.

Reply:
Yes when the screen gets small, the stepper will be vertical instead of horizontal